### DISCUSSION TITLE
blog/coming-soon/

### DISCUSSION THREAD

**Discussion by giscus** [ekrof/ekerhovd.it#5] [Announcements]
# blog/coming-soon/

 1 2 3 4 5 6 7 8 &lt;section id=&#34;main&#34;&gt; &lt;div&gt; &lt;h1 id=&#34;title&#34;&gt;{{ .Title }}&lt;/h1&gt; {{ range .Pages }} {{ .Render &#34;summary&#34;}} {{ end }} &lt;/div&gt; &lt;/section&gt; 

http://localhost:1313/blog/coming-soon/

**Comment by ekrof** [comment]
My first comment!
